inst/components/scss/sidebar.scss

```diff
@@ -244,7 +255,7 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sideba
       color: var(--_main-fg);
       // The CSS variable (set via JS) is here to help avoid overlapping toggles
       top: calc(
-          var(--bslib-sidebar-overlap-counter, 0) *
+          var(--_js-counter-overlap, 0) *
```

Comment:
This is partly my fault, but I'm not loving this name (what is this counting?).  Maybe `--_toggle-overlap-js-counter` would be better?

Reply:
Agreed, this could be clearer. I'd like the "js" to come first, but otherwise I'm totally in favor of using a more descriptive name.